dynamic_programming/tree_height.cpp

```diff
@@ -0,0 +1,60 @@
+/// C++ Program to find height of the tree using bottom - up DP.
```

Comment:
Why /// instead of // ??

Reply:
Noted.
Actually in my IDE it gives better commenting visuals compare to //.

Reply:
What IDE do you use?

Reply:
I prefer to use CodeBlocks

Reply:
![Capture_1](https://user-images.githubusercontent.com/44433995/79600994-ba619b80-8105-11ea-8c77-28720449d9ce.JPG)
![Capture_2](https://user-images.githubusercontent.com/44433995/79600998-bb92c880-8105-11ea-9141-05795e6cec44.JPG)